src/test/suite/fixtures/recorded/surroundingPair/html/clearBoundsOx2.yml

```diff
@@ -0,0 +1,41 @@
+languageId: plaintext
```

Comment:
Ok looks great. One more nit: would be good to put the plaintext tests in the other dir of text-based surrounding pair tests